Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub_test.py

```diff
@@ -166,3 +167,24 @@ def test_fetch_incidents(mocker):
     client = MockClient()
     fetch_incidents(client, 'Low', False, None)
     assert demisto.setLastRun.call_args[0][0]['lastRun'] == '2020-03-22T13:22:13.934000+00:00'
+
+
+def test_list_members_command(mocker):
+    """
+    Given:
+        - mock aws_session
+    When:
+        - Running list_members_command
+    Then:
+        - Ensure that the command was executed correctly. In particular, ensure that the datetime fields are convereted to str.
+    """
+    aws_client = AWSClient("reg", "", "", 900, "p", "mock_aws_access_key_id", "mock_aws_secret_access_key", "", "", 3)
+    client = aws_client.aws_session(service='securityhub', region="reg", role_arn='roleArnroleArnroleArn',
+                                    role_session_name='roleSessionName')
+    now_time = datetime.datetime.now()
```

Comment:
For consistency between rubs, let's give here a constant time (not now). Something like this:
```suggestion
    time_val = dateparser.parse('2022-01-01',settings={'TIMEZONE': 'UTC'})
```